src/actor/Verifier.spec.ts

```diff
@@ -63,13 +63,12 @@ describe('Verifier', () => {
     }
 
     blockchainApi.query.attestation.attestations.mockReturnValue(
-      new Option(
-        Tuple,
-        new Tuple(
-          [H256, AccountId, Text, Bool],
-          [cType.hash, attester.getAddress(), undefined, false]
-        )
-      )
+      new Option(Tuple.with([H256, AccountId, 'Option<H256>', Bool]), [
+        '0x1234', // ctype hash
```

Comment:
We should try to only use real hashes, and not these mocks, as this will be decoded to something unequal to 0x1234

Reply:
I DID IT! 06e712f069524cbf1dd4ca1e8ebab7d63a4b996d